docs/samediff/templates/model-import.md

```diff
@@ -1 +1,62 @@
-# Getting started: importing TensorFlow and ONNX models into SameDiff
\ No newline at end of file
+---
+title: Getting started: importing TensorFlow and ONNX models into SameDiff
+short_title: Model import
+description: importing TensorFlow and ONNX models into SameDiff
+category: SameDiff
+weight: 3
+---
+
+# Getting started: importing TensorFlow models into SameDiff
+
+## What models can be imported into samediff
+
+Currently samediff supports the import of Tensorflow frozen graphs through the various Samediff.importFrozenTF methods. 
+Tensorflow documentation on frozen models can be found [here](https://www.tensorflow.org/guide/saved_model#the_savedmodel_format_on_disk). 
+
+    import org.nd4j.autodiff.samediff.SameDiff;
+    
+    SameDiff sd = SameDiff.importFrozenTF(modelFile);
+    
+ ## Finding the model input/outputs and running inference
+ 
+ After you import the Tensorflow model there are 2 ways to find the inputs and outputs. The first method is to look at the output of
+ 
+     sd.summary();
+     
+ Where the input variables are the output of no ops, and the output variables are the input of no ops.  The other way to find the inputs and outputs is
+ 
+      List<String> inputs = sd.inputs();
+      List<String> outputs = sd.outputs());
+    
+ To run inference use:
+ 
+    INDArray out = sd.batchOutput()
+        .input(inputs, inputArray)
+        .output(outputs)
+        .execSingle();
+
+##  Node skipping, import overrides and model validation.
```

Comment:
Let's split this up into two sections: the former many users might consider using. The latter: not many users should need.
`## Import Validation` + provide simple example
`## Advanced: Node Skipping and Import Overrides`